contents/docs/self-host/deploy/upgrade-notes.md

```diff
@@ -115,3 +115,20 @@ If you didn’t make any customization to those, there’s nothing you need to d
 
 - drops support for Kubernetes 1.19 as it has reached end of life on 2021-10-28
 - adds support for Kubernetes 1.23 released on 2021-12-07
+
+### Upgrading from 9.x.x
+
+10.0.0 introduces two major changes:
+
+1. as of today we've been including additional `StorageClass` definition into our Helm chart when installing it on AWS or GCP platforms. Starting from this release, we will not do that anymore and we will rely on the cluster default storage class. If you still want to install those additional storage classes, simply set `installCustomStorageClass: true` in your `values.yaml`. If you are planning to use the default storage class, make sure you are running with our [requirement settings](https://posthog.com/docs/self-host/deploy/aws#cluster-requirements) (`allowVolumeExpansion` set to `true` and `reclaimPolicy` set to `Retain`).
+
+2. we have renamed few chart inputs in order to reduce confusion and align our naming convention to the industry standards:
+
+    - `clickhouseOperator.enabled` -> `clickhouse.enabled`
```

Comment:
Can we raise an error within helm if we have and `clickhouseOperator` keys @guidoiaquinti ?

Reply:
We could but I don't know if we should 🤔

Reply:
Idea I had in the past not sure that would be good either: we can make helm fail major upgrades preemptively (similarly to the right file used verification) if they don't provide a value `majorUpgrade=true` (e.g. via `--set majorUpgrade=true`) at the end of the upgrade command & link folks to the upgrade notes.